src/globals.d.ts

```diff
@@ -1,4 +1,5 @@
 declare const __PACKAGE_NAME__: string;
 declare const __PACKAGE_VERSION__: string;
 declare const __PC_DEV_ENVS__: string[];
+declare const __GC_DEV_EXTRA_ENVS__: unknown[];
```

Comment:
Would be great if we could declare this as `Environment[]`.

I played with this a bit and it's a bit unfortunate that adding an import then registers this as a module; thereby requiring that we export these declarations and then import them into the consuming files.

[this SO article](https://stackoverflow.com/questions/45420448/how-to-import-external-type-into-global-d-ts-file) suggests that we might be able to use a dynamic import to get around this issue.

Maybe you can take a look and see if there's a solution that we can type this correctly and not import everywhere?

If this turns into a research project or overly inflates this PR, we can leave it as unknown.  but, work a small look I think.

Reply:
nice!  great to have the type declared here!